polaris-migrator/src/utilities/sass.ts

```diff
@@ -376,18 +404,49 @@ export function createSassMigrator(name: string, ruleFn: PolarisMigrator) {
 
         for (const report of reportsForNode) {
           node.before(
-            report.severity === 'suggestion'
-              ? createInlineComment(report.message)
```

Comment:
Suggestions are now inserted as part of the walker function, so this code can remove the conditional.